hw/ip/otbn/dv/otbnsim/stepped.py

```diff
@@ -405,9 +405,10 @@ def on_step_crc(sim: OTBNSim, args: List[str]) -> Optional[OTBNSim]:
     return None
 
 
-def on_send_lc_escalation(sim: OTBNSim, args: List[str]) -> Optional[OTBNSim]:
-    check_arg_count('send_lc_escalation', 0, args)
-    sim.on_lc_escalation()
+def on_send_err_escalation(sim: OTBNSim, args: List[str]) -> Optional[OTBNSim]:
+    check_arg_count('on_send_err_escalation', 1, args)
```

Comment:
The first argument here is the name of the command that was sent on stdin, so you don't want the "on_" prefix.